src/SFA.DAS.Authorization/SFA.DAS.Authorization.csproj

```diff
@@ -26,6 +26,7 @@
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-19270-01" PrivateAssets="All" />
         <PackageReference Include="NLog" Version="4.5.11" />
         <PackageReference Include="NLog.Extensions.Logging" Version="1.4.0" />
+        <PackageReference Include="Scrutor" Version="3.1.0" />
```

Comment:
are we using this nuget package, if so why?

Reply:
yep, to use decorate. Decorate - used to decorate already registered services .
place of usage is :   services.Decorate<IAuthorizationService, AuthorizationServiceWithDefaultHandler>();